content/en/docs/collector/deployment/best-practices.md

```diff
@@ -0,0 +1,189 @@
+description: How to configure collectors for various use cases
+weight: 4
+---
+
+Now that you are equipped with the essential deployment patterns for the
+collector, let's have a closer look at best practices for collector (pipeline)
+configurations for different use cases.
+
+## Fan Out
+
+Export signals into more than one backend. For example, for testing, policy
+(signals from dev/test go into backend X, prod goes into Y), or migrations from
+one back-end to another (cut-over) or different signal types go into different
+backends.
+
+<!-- prettier-ignore-start -->
+{{< tabpane persistLang=false >}}
+{{< tab header="Per type" lang="yaml" >}}
+receivers:
+  otlp:
+    protocols:
+      grpc:
+
+exporters:
+  jaeger:
+    endpoint: "https://jaeger.example.com:14250"
+    insecure: true
+  prometheusremotewrite:
+    endpoint: "https://prw.example.com/v1/api/remote_write"
+
+service:
+  pipelines:
+    metrics/prod:
+      receivers: [otlp]
+      processors: [batch]
+      exporters: [prometheusremotewrite]
+    traces:
+      receivers: [otlp]
+      processors: [batch]
+      exporters: [jaeger]
+{{< /tab >}}
+{{< tab header="Per environment" lang="yaml" >}}
+receivers:
+  otlp:
+    protocols:
+      grpc:
+
+processors:
+  metricstransform/dev:
+    transforms:
+    - include: ".*"
+      match_type: regexp
+      action: update
+      operations:
+      - action: delete_label_value
+        label: env
+        label_value: prod
+
+exporters:
+  prometheus:
+    endpoint: "0.0.0.0:1234"
+    namespace: dev
+    metric_expiration: 30m
+    enable_open_metrics: true
+    resource_to_telemetry_conversion:
+      enabled: true
+  prometheusremotewrite:
+    endpoint: "https://prw.example.com/v1/api/remote_write"
+
+service:
+  pipelines:
+    metrics/prod:
+      receivers: [otlp]
+      exporters: [prometheusremotewrite]
+    metrics/dev:
+      receivers: [otlp]
+      processors: [metricstransform/dev]
+      exporters: [prometheus]
+{{< /tab >}}
+{{< tab header="Migration" lang="yaml" >}}
+receivers:
+  otlp:
+    protocols:
+      grpc:
+
+exporters:
+  prometheusremotewrite/old:
+    endpoint: "https://old.example.com/v1/api/remote_write"
+  prometheusremotewrite/new:
+    endpoint: "https://new.example.com/v1/api/remote_write"
+
+service:
+  pipelines:
+    metrics/prod:
+      receivers: [otlp]
+      exporters: [prometheusremotewrite/old]
+    metrics/target:
+      receivers: [otlp]
+      exporters: [prometheusremotewrite/new]
+{{< /tab >}}
+{{< /tabpane>}}
+<!-- prettier-ignore-end -->
+
+## Normalizing
+
+Normalize the metadata from different instrumentations
+
+```yaml
+receivers:
+  prometheus:
+    config:
+      scrape_configs:
+        - job_name: 'otel-collector'
+          scrape_interval: 5s
+          static_configs:
+            - targets: ['0.0.0.0:8888']
+  jaeger:
+    protocols:
+      grpc:
+      thrift_binary:
+      thrift_compact:
+
+processors:
+  attributes:
+    actions:
+      - key: cluster
```

Comment:
We should make sure that we use attributes covered by the semantic conventions across the documentation.

Reply:
Makes sense and also note that as agreed, removing the best practices for this PR. Will address in the next PR.